content/docs/getting-started/getting-started-for-admins/index.md

```diff
@@ -35,10 +35,18 @@ If you wish to tightly restrict the access that Roadie has, please [follow this
 
 ### Step 1. Create a Personal Access Token (PAT)
 
-Visit the [developer settings of your GitHub account](https://github.com/settings/tokens) and create a PAT which has the scopes `repo`, `user`, `read:org` and `read:discussion` .
+Visit the [developer settings of your GitHub account](https://github.com/settings/tokens) and create a PAT which has the scopes `repo`, `user`, `read:org` and `read:discussion`.
 
 The official GitHub docs for creating a Personal Access Token are [here](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).
 
+---
```

Comment:
I find that the extra heading and dividers breaks the flow a little when reading.

What about [this pattern](https://deploy-preview-150--roadie.netlify.app/docs/getting-started/technical-documentation/#step-1-add-the-mkdocs-configuration-file) for highlighting a paragraph?

![Screenshot 2021-03-23 at 08 08 25](https://user-images.githubusercontent.com/562403/112113632-f8393080-8bae-11eb-83b8-50b193ded950.png)